### ISSUE TITLE
Release FAS model and CASE dataset on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [24DavidHuang/FAS#3]
Hi @24DavidHuang 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Arxiv (https://arxiv.org/abs/2609.04236) and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://arxiv.org/abs/2609.04236

The paper page lets people discuss about your paper and lets them find artifacts about it, such as your models or dataset. You can also claim the paper as yours, which will show up on your public profile at Hugging Face, and add GitHub and project page URLs.

It would be great to make the FAS checkpoint and the CASE dataset available on the 🤗 Hub, to improve their discoverability and visibility. We can add metadata tags so that people can find them when filtering https://huggingface.co/models and https://huggingface.co/datasets.

## Uploading the model

See our model upload guide here: https://huggingface.co/docs/hub/models-uploading.

Since your model is built with PyTorch, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class, which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, you can directly upload the `.pth` checkpoint and add usage examples in the model card.

We recommend uploading each checkpoint as a separate model repository, so that download statistics and tags work correctly. Afterward, we can link the model to the paper page.

## Uploading the dataset

Would be awesome to make the CASE dataset available on Hugging Face so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/CASE")
```

See the dataset upload guide: https://huggingface.co/docs/datasets/loading.

The [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) will also allow people to explore the audio samples and labels directly in the browser. Since the dataset is currently hosted on Google Drive, moving it to Hugging Face would make it much easier for the community to download and use.

Let me know if you're interested or if you need any guidance — I'd be happy to help you step by step!

Kind regards,

Niels  
ML Engineer @ Hugging Face 🤗
